src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = dailyCost * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify the conditions. Since you're using `return` inside the `if` blocks, you can remove the `else` keyword to make the code cleaner.